templates/pages/index.html.njk

```diff
@@ -4,22 +4,71 @@
 
 {% block header %}{% endblock %}
 
-{% block content %}
-  <section class="tc grid mb6 mb7-l">
-    <div class="grid-item-center">
-      {{ svgIcon(contents, 'jsconf-logo-full', width = 564, height = 84, alt = 'JSConf EU') }}
+{% block main %}
+  <section class="section-intro">
+    <div class="section-intro-background">
+      <div class="grid">
+        <div class="grid-item-full">
+          <div class="section-intro-festival">
+            {{ svgIcon(contents, 'festival', width = 520, height = 124, alt = 'festival') }}
+          </div>
+          <div class="section-intro-logo">
+            {{ svgIcon(contents, 'jsconf-logo-white-full', width = 258, height = 39, alt = 'JSConf EU') }}
+          </div>
+        </div>
+        <p class="grid-item-full">
+          {{ fullConferenceDate }} &mdash; Berlin, Germany
+          <br>
+          JSConf EU is the labour-of-love conference for the JavaScript community in Europe.
+        </p>
+        <p class="grid-item-full">
+          <a href="{{ buyTicketsLink }}" class="btn btn--white btn--center">
+            Buy ticket
+          </a>
+        </p>
+      </div>
     </div>
-    <p class="grid-item-center">
-      {{ fullConferenceDate }} &mdash; Berlin, Germany
-      <br>
-      JSConf EU is the labour-of-love conference for the JavaScript community in Europe.
-      <br>
-      Our 2019 homepage is currently under construction. Please check back soon.
-    </p>
-    <p class="grid-item-center">
-      <a href="{{ buyTicketsLink }}" class="btn buy-ticket">
-        Buy ticket
-      </a>
-    </p>
   </section>
+  <a class="section-video" href="#" target="_blank">
+    <div class="section-video-icon">
+      {{ svgIcon(contents, 'play', width = 86, height = 86, alt = 'Play') }}
+    </div>
+    <div class="section-video-text ttu mb4 tc">Impressions of JSConf EU 2018</div>
+  </a>
+  <section class="section-cfs">
+    <div class="grid">
+      <div class="grid-item-full">
+        <div class="section-ornament section-ornament--right">
+          <h2 class="heading--massive heading--section">Call() for Speakers</h2>
+          <p class="mr6-l">
+            Our call for speakers is open until December 23, 2018. lorem ipsum dolor sit lalala orem ipsum dolor sit
+            lalala orem ipsum dolor sit lalala orem ipsum dolor sit lalala orem ipsum dolor sit lalalaorem ipsum dolor
+            sit lalala.
+            <a href="{{ contents['call-for-speakers.json'].url }}" class="link--slateblack">Read more</a>
+          </p>
+        </div>
+      </div>
+    </div>
+  </section>
+  <div class="section-news">
```

Comment:
We should make this dynamic, added it as static text just for the preview.

Reply:
Please wrap in comments for now. I'll add logic at a later time.